server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -202,11 +202,12 @@ public enum MetadataVersion {
     // Add new fetch request version for KIP-951
     IBP_3_7_IV4(19, "3.7", "IV4", false),
 
-    // Add ELR related supports (KIP-966).
-    IBP_3_8_IV0(20, "3.8", "IV0", true),
+    // New version for the Kafka 3.8.0 release.
+    IBP_3_8_IV0(20, "3.8", "IV0", false),
 
+    // Add ELR related supports (KIP-966).
```

Comment:
I wonder if we should put a comment here explaining that the MV below this point are not production ready. We could also add a few words explaining that features must be reviewed and they must moved to the next one if they are not ready when the next MV is made production ready.

Reply:
I will add a comment.